uvicorn/protocols/http/h11_impl.py

```diff
@@ -543,10 +532,11 @@ async def send(self, message: "ASGISendEvent") -> None:
 
     async def receive(self) -> "ASGIReceiveEvent":
         if self.waiting_for_100_continue and not self.transport.is_closing():
-            event = h11.InformationalResponse(
-                status_code=100, headers=[], reason="Continue"
-            )
-            output = self.conn.send(event)
+            # fmt: off
+            headers: List[Tuple[str, str]] = []
+            informational_response = h11.InformationalResponse(status_code=100, headers=headers, reason="Continue") # noqa: E501
+            output = self.conn.send(event=informational_response)
+            # fmt: on
```

Comment:
I don't really mind the `event` variable (i.e. instead of `informational_response`). It feels a little verbose to specify the type of the response in the variable name (since there is type inference here). You're of course welcome to use whatever style makes you happy here though :)

These fmt on/off E501 ignores are sad to see. Why not split the response construction over multiple lines? Maybe you _should_ bump the line length for the project, although 114 chars is pretty long.

Reply:
Ah, I thought we were already using the `event` variable name in this scope. I've changed for `event` once again.